packages/frontend/src/components/BidsList/BidsList.tsx

```diff
@@ -24,28 +24,33 @@ export const BidsList = ({ bids, view = 'full', isLoadingParams }: Props) => {
   }, [userBid, auctionWinnersCount])
 
   return (
-    <BidList>
-      {isLoadingParams ? (
-        emptyBids.map((emptyBid) => <EmptyBidListEntry key={emptyBid} place={emptyBid} />)
-      ) : (
-        <>
-          {bids.map((bid) => (
-            <BidListEntry
-              key={bid.bidderAddress}
-              bid={bid}
-              isUser={userBid && addressEqual(userBid.bidderAddress, bid.bidderAddress)}
-              view={view}
-            />
-          ))}
-          {userRaffleBid && view === 'short' && (
-            <>
-              <Separator color={Colors.Grey} />
-              <BidListEntry bid={userRaffleBid} isUser view={view} />
-            </>
-          )}
-        </>
+    <>
+      <BidList>
+        {isLoadingParams ? (
+          emptyBids.map((emptyBid) => <EmptyBidListEntry key={emptyBid} place={emptyBid} />)
+        ) : (
+          <>
+            {bids.map((bid) => (
+              <BidListEntry
+                key={bid.bidderAddress}
+                bid={bid}
+                isUser={userBid && addressEqual(userBid.bidderAddress, bid.bidderAddress)}
+                view={view}
+              />
+            ))}
+            {userRaffleBid && view === 'short' && (
+              <>
+                <Separator color={Colors.Grey} />
+                <BidListEntry bid={userRaffleBid} isUser view={view} />
+              </>
+            )}
+          </>
+        )}
+      </BidList>
+      {view === 'short' && !isLoadingParams && (
+        <BidListText>You’re taking part in the {userRaffleBid ? 'raffle' : 'auction'}!</BidListText>
       )}
```

Comment:
Looks like it will tell the user that they're taking part in the auction when they don't have any bids placed, the component probably needs to check if a user bid exists in the first place before displaying this